src/env.ts

```diff
@@ -18,6 +18,14 @@ export const env = createEnv({
     GOOGLE_CLIENT_SECRET: z.string().optional(),
 
     ENCRYPTION_KEY: z.string().min(30),
+
+    // R2
+    // Don't include the bucket name in the URL
+    R2_URL: z.string().min(1),
+    R2_PUBLIC_URL: z.string().min(1),
+    R2_BUCKET: z.string().min(1),
+    R2_ACCESS_KEY: z.string().min(1),
+    R2_SECRET_ACCESS_KEY: z.string().min(1),

```

Comment:
Make all optional. If user doesn't specified it, when upload, our API will return error saying that "Server does not support file upload"

Reply:
Done